File: src/com/cs/model/solaire/Batterie.cs
using System;
namespace src.com.cs.models {
public class Batterie {
 {
    private int idbatterie;
    private string nombatterie;
    private object capacite;

    public Batterie(int idbatterie, string nombatterie, object capacite) {
        this.idbatterie = idbatterie;
        this.nombatterie = nombatterie;
        this.capacite = capacite;
    }
    public int Idbatterie {
        get { return idbatterie; }
        set { idbatterie = value; }
    }
    public string Nombatterie {
        get { return nombatterie; }
        set { nombatterie = value; }
    }
    public object Capacite {
        get { return capacite; }
        set { capacite = value; }
    }
}
    }
}

File: src/com/java/model/solaire/Batterie.java
package com.java.model.solaire;

import java.lang.Integer;
import java.lang.Object;
import java.lang.String;

public class Batterie {
  private Integer idbatterie;

  private String nombatterie;

  private Object capacite;

  public Batterie(Integer idbatterie, String nombatterie, Object capacite) {
    this.idbatterie = idbatterie;
    this.nombatterie = nombatterie;
    this.capacite = capacite;
  }

  public Integer getIdbatterie() {
    return idbatterie;
  }

  public void setIdbatterie(Integer idbatterie) {
    this.idbatterie = idbatterie;
  }

  public String getNombatterie() {
    return nombatterie;
  }

  public void setNombatterie(String nombatterie) {
    this.nombatterie = nombatterie;
  }

  public Object getCapacite() {
    return capacite;
  }

  public void setCapacite(Object capacite) {
    this.capacite = capacite;
  }
}
